guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -58,40 +58,14 @@ A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selec
 ifdef::katello,satellite,orcharhino[]
 . In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
+. Optional: Select the *Insecure* option, if you want to make the registration call insecure.
```

Comment:
I would rephrase it:
It's not like users want to make it insecure. It's because the CA file is not in the host's "trusted store."
So I would say something like, "If the CA is not trusted by the host, use insecure for the initial call. Then, the registration will download the CA automatically and use a secure connection for the next calls and communication."

Reply:
@stejskalleos That's exactly the situation I'm trying to prevent by having the user to configure the host before registration. The pre-registration config involves adding the CA cert to the trust store. Are there any other use cases for an insecure call?

Reply:
If users SSH into a host to run the curl/wget command, then they'd also have a way to securely deploy the Foreman CA cert. With guides/common/modules/proc_configuring-a-host-for-registration.adoc, which I like, there should not be a valid use case for `Insecure` anymore.

I also think the explanation by Leos makes sense; it's IMHO better than "click insecure to make X insecure" -> "Select insecure if your host does not trust the Foreman CA prior to the registration".